mbtb_app/web_portal/mbtb_portal/api/controllers/admin/suspend-user-form.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+
+
+  friendlyName: 'Suspend user form',
+
+
+  description: '',
+
+
+  inputs: {
+
```

Comment:
5 -> We'll receive request_ids here from `assets/js/suspend_user_form.js` file, so add it like
```
requests_ids:{
      type: 'json',
      required: true,
      description: 'Receives selected ids from admin for patch request'
    }
```